igvc_rviz_plugins/src/sensor_panel.cpp

```diff
@@ -0,0 +1,109 @@
+#include <igvc_rviz_plugins/sensor_panel.h>
+#include <pluginlib/class_list_macros.h>
+#include <QVBoxLayout>
+
+/*
+ * Just as in the header, everything needs to happen in the igvc_rviz_plugins namespace.
+ */
+namespace igvc_rviz_plugins
+{
+sensor_panel::sensor_panel(QWidget *parent) : rviz::Panel(parent)  // Base class constructor
+{
+  // Panels are allowed to interact with NodeHandles directly just like ROS nodes.
+  ros::NodeHandle nh;
+
+  // Initialize all labels for displaying some data
+  labels["IMU"] = new QLabel("IMU: Placehold\n");
+  labels["Lidar"] = new QLabel("Lidar: Placehold\n");
+  labels["Center Camera"] = new QLabel("Center Camera: Placehold\n");
+  labels["GPS"] = new QLabel("GPS: Placehold\n");
+
+  isActive = new bool [labels.size()];
+
+  /*
+  *Makes timer that repeats every interval seconds
+  *When timer goes off, it calls the timer method
+  */
+  sensor_timer = new QTimer();
```

Comment:
shared pointer

Reply:
I tried using shared pointers and ended up getting some boost errors. This may take a bit, but once I fix this everything should be working fine.

Reply:
If shared pointers are giving issues then just leave them as is.

Reply:
Ok, I'll get rid of them. I also realized that QT doesn't like it when I use shared pointers either.